### DISCUSSION TITLE
✅ Generating ssh keys

### DISCUSSION THREAD

**Discussion by amir-fo** [isontheline/pro.webssh.net#832] [Q&A] [answered]
Thank you for an amazing app. I was wondering if theres an option to generate ssh keys on app? I couldnt find anything messing around the app. If theres not, is there any possiblity you could add this feature?

**Comment by isontheline** [comment, marked as answer]
Hello @amiroof 👋

Thank you so much for your support 🙏
Key Generation is planned here : https://github.com/isontheline/pro.webssh.net/issues/46

I think I will release a first beta before the end of April.

**Reply by amir-fo** [reply]
This app is already perfect in all aspects. From the developer to the minor stuff. Thank you for everything.

**Reply by isontheline** [reply]
Thank you so much @amiroof 🙏

Please subscribe to #46 in order to receive notifications about the beta release availability that should occurs before end of April.

Have a noce day ☀️
